spec/content/automate/ManageIQ/AutomationManagement/AnsibleTower/Operations/Methods.class/__methods__/available_credentials_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 describe ManageIQ::Automate::AutomationManagement::AnsibleTower::Operations::AvailableCredentials do
   let(:ansible_manager) { FactoryBot.create(:automation_manager_ansible_tower) }
```

Comment:
@lfu Should this now be `embedded_automation_manager_ansible` or `provider_embedded_ansible`?